src/gtfs.rs

```diff
@@ -66,6 +72,11 @@ impl TryFrom<RawGtfs> for Gtfs {
             (*fare_rules.entry(f.fare_id.clone()).or_default()).push(f);
         }
 
+        let mut fare_products = HashMap::<String, Vec<FareProduct>>::new();
+        for f in raw.fare_products.unwrap_or_else(|| Ok(Vec::new()))? {
+            (*fare_products.entry(f.id.clone()).or_default()).push(f);
```

Comment:
I know that the existing code juste above uses this useless deref.
Either you fix it, or leave it as it is.

```suggestion
            fare_products.entry(f.id.clone()).or_default().push(f);
```